src/style.css

```
@@ -1 +1,62 @@
-/* Styles go here */
+html {
+  font-family: Roboto, serif;
+  font-size: 12px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

```suggestion
* {
  box-sizing: border-box;
}
```
redundant to reset margins and paddings for all selectors.
reset for elements which really need it

---

@VitaliyBondarenko1982  
if I remove the margin and padding, my tests break, I have the task to remove the default indents

---

not remove, reset just for elements which really need (body, ul, h1, h2..., p)